tests/unit/test_lexer.py

```diff
@@ -229,7 +229,7 @@ def testMultilineBlockName(self):
         tokens = self.lexer.tokenize(text)
         self.assertEqual(
             tokens, [
-                ('long', ' \\\n ', 'bloc\\\n name\\\n     \\\n'), '\n', 'long'
+                ('long', ' \\\n ', 'bloc name '), '\n', 'long'
```

Comment:
Weirdly enough, this trailing space does match the Perl Config::General behavior. The same string gets parsed into:
```
{ 'long' => { 'bloc name ' => {}}};
```

Reply:
I assume we are going to live with it...